cocos/core/pipeline/render-additive-light-queue.ts

```diff
@@ -145,10 +152,7 @@ export class RenderAdditiveLightQueue {
             const ro = this._renderObjects[i];
             const model = ro.model;
             const subModels = model.subModels;
-
-            // this assumes light pass index is the same for all submodels
-            const lightPassIdx = getLightPassIndex(subModels);
-            if (lightPassIdx < 0) continue;
+            if (!getLightPassIndices(subModels, _lightPassIndices)) continue;
```

Comment:
This doesn't seem right, (!-1) is false, (!0) is true

Reply:
the return value is a single bool